src/Sentry.Unity/SentryUnitySDK.cs

```diff
@@ -28,6 +28,8 @@ private SentryUnitySdk(SentryUnityOptions options)
             return null;
         }
 
+        MainThreadData.CollectData();
```

Comment:
Do they sync to native when moved here?

Reply:
Yes. For our purposes the `MainThreadData` can be considered immutable and needs to be collected and then synched once. The sync to native happens after initialization
https://github.com/getsentry/sentry-unity/blob/f1fe17a033b2a0861c937efd931dd19645908bda/src/Sentry.Unity/SentryUnitySDK.cs#L70-L77